### ISSUE TITLE
Incorrect legend postion in subplot 

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#1590]
```
gmt begin map 
    gmt subplot begin 1x2 -Fs7c/5c -A

    gmt subplot set 0
    gmt basemap -R0/10/0/10 -Baf
    gmt legend -DjRT+w1.2c -F+p0.5p << EOF
S 0.1c c 0.1c red - 0.3c a
S 0.1c s 0.1c blue - 0.3c b
EOF

    gmt subplot set 1 -Cw1c
    gmt basemap -R0/10/0/10 -Baf
    gmt legend -DjRT+w1.2c -F+p0.5p << EOF
S 0.1c c 0.1c red - 0.3c a
S 0.1c s 0.1c blue - 0.3c b
EOF
    gmt subplot end
gmt end show
```
![image](https://user-images.githubusercontent.com/3974108/65292482-9fa25680-db25-11e9-9df5-d2f9aa40b359.png)

**Comment by PaulWessel** [comment]
So the content of that legend is misplaced but the outline is correct.
I assume you are OK with the position of b) since that should not be affected by -C.

**Comment by seisman** [comment]
Yes, I'm OK with the position of (b).

**Comment by seisman** [comment]
Closed by #1667.
